js/foundation.orbit.js

```diff
@@ -368,11 +370,12 @@ class Orbit extends Plugin {
           this.timer.restart();
         }
       }
-    /**
-    * Triggers when the slide has finished animating in.
-    * @event Orbit#slidechange
-    */
-      this.$element.trigger('slidechange.zf.orbit', [$newSlide]);
+     /**
+      * Triggers when the slide has finished animating in.
+      * does'nt realy make any sense, keep event for compatibility reasons.
```

Comment:
Not sure what you nean that it does not make any sense. We can get the index and wait for onanimation/transitionend to work with this.

Reply:
> slidechange.zf.orbit | Triggers when the slide has finished animating in.

Thats wrong. is triggert after animation is started and right after beforeslidechange.

Anyway, i had some proposals. Think about it and use it according to your wish. ;)

Reply:
Then we should change it in the docs. Please do not mix multiple issues in this PR so we can fix one issue after another and this makes reviews much easier.